package.json

```diff
@@ -62,6 +62,7 @@
     "mongoose": "^5.9.16",
     "mongoose-sequence": "^5.2.2",
     "nconf": "^0.10.0",
+    "ngx-matomo": "^0.1.4",
```

Comment:
Have you tested this successfully? I see in [the docs](https://www.npmjs.com/package/ngx-matomo) that it supports "Angular 5, 6, 7 & 8" but we are on Angular 1 so I'm curious how functional this is.

I had seen https://github.com/angulartics/angulartics as an alternative (or maybe https://github.com/mike-spainhower/angular-piwik). Lemme know what you think.

Reply:
So to test this, I need the piece of code to uploaded on the url. I saw an example of this being used for AngularJS somewhere, so I gave it a shot. If not I can switch to angulartics.

Reply:
Is there any way for you to test it locally on your computer first? If you could get data going to your Matomo dashboard that shows page views from your local computer, then this will work great! I'd just rather not merge something if we're not sure whether it works.

Let me know if you need help setting this up. Matthew or John may be able to help too.

Reply:
Hey @cooperka. I was able to test the data collection locally. Here's a screenshot
<img width="1156" alt="Screenshot 2020-11-03 at 2 09 52 PM" src="https://user-images.githubusercontent.com/28644407/98030022-5e7b0d80-1dde-11eb-9c7d-9324ac3a2101.png">